### DISCUSSION TITLE
The server did not understand your request

### DISCUSSION THREAD

**Discussion by ThaFacialHair** [Motion-Project/motion#1750] [Q&A]
After Updating to the newest version and getting my conf files all set back up I now get a static page of "The server did not understand your request" when going to the stream address. I can provide any further configs needed for diagnostics purposes.

**Comment by tosiara** [comment]
Which url do you use?

**Reply by Cyberjakkes** [reply]
I have same issue, with internal ip (192.168.*.*) and with port 8081 at the end. That was working good wit 1 camera but after adding 2 camera's it not working anymore. port 8080 shows al camera's

**Reply by Cyberjakkes** [reply]
I found a solution, after the port you have to add a number from the camera http://192.168.*.*:8081/2/stream, and then it works
